pkg/controllers/uiplugin/compatibility_matrix_test.go

```diff
@@ -242,6 +248,46 @@ func TestLookupImageAndFeatures(t *testing.T) {
 			expectedFeatures: []string{"acm-alerting"},
 			expectedErr:      nil,
 		},
+		{
+			pluginType:       uiv1alpha1.TypeMonitoring,
+			clusterVersion:   "v4.19",
+			acmVersion:       "v2.11.3",
+			expectedKey:      "ui-monitoring",
+			expectedFeatures: []string{"acm-alerting", "perses-dashboards"},
+			expectedErr:      nil,
+		},
+		{
+			pluginType:       uiv1alpha1.TypeMonitoring,
+			clusterVersion:   "v4.19.0-0.nightly-2024-06-06-064349",
+			acmVersion:       "v2.11.3",
+			expectedKey:      "ui-monitoring",
+			expectedFeatures: []string{"acm-alerting", "perses-dashboards"},
+			expectedErr:      nil,
+		},
+		{
+			pluginType:       uiv1alpha1.TypeMonitoring,
+			clusterVersion:   "v4.19",
```

Comment:
The cluster version for Perses could be lower as in older versions Perses dashboards can be added as a new Menu item. Nothing to block this PR as this can be adjusted when we decide.

Reply:
Alright, I'll leave it as is for now.

Reply:
The latest push allows "perses-dashboards" feature flag to be added on OCP 4.14+.